platform/security/encryption/platform-security-encryption-impl/src/main/java/ddf/security/encryption/impl/EncryptionServiceImpl.java

```diff
@@ -117,9 +117,12 @@ public String decryptValue(String wrappedEncryptedValue) {
 
   @Override
   public String encryptValue(String unwrappedPlaintext) {
-    if (unwrappedPlaintext == null || unwrappedPlaintext.equals("")) {
+    if (unwrappedPlaintext == null) {
```

Comment:
❓ Can you do something like `if (unwrappedPlaintext.isEmpty()) return unwrappedPlaintext;` ?
http://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isEmpty(java.lang.String)

Reply:
there needs to be two separate things returned if null or "" and isempty() would instead catch both of those. based on comment made on other PR https://github.com/codice/ddf/pull/4083#discussion_r239947550.

Reply:
I'm confused it looks like you're returning whatever the value of unwrappedPlaintext is for both scenarios:

null -> return null
"" -> return ""

It won't work to return it?

Reply:
ah, sorry i was confused. yes you're correct good suggestion

Reply:
I originally used StringUtils, but realized I was bringing in Apache Commons Lang just to check for an empty string. I can certainly return the input. I also update `decryptValue` to do the same.

Reply:
So, why no not keep the test as it was before and return `unwrappedPlaintext`? Same in `decryptValue()` above?